New design guide: Securing guest wireless networks (#18820)
cloudflare/cloudflare-docs: src/content/docs/reference-architecture/design-guides/securing-guest-wireless-networks.mdx

In order for cloudflare to provide network level filtering, Cloudflare must be in the traffic path for more than just the DNS request. This is achieved by routing Internet-bound traffic over an [IPsec](https://www.cloudflare.com/learning/network-layer/what-is-ipsec/) tunnel to Cloudflare. Cloudflare's [Magic WAN](/magic-wan/) service allows 3rd party devices to establish IPsec or GRE tunnels to the Cloudflare network. It's also possible to just deploy our [Magic WAN Connector](/magic-wan/configuration/connector/), a pre-configured lightweight network appliance that automatically creates the tunnel back to Cloudflare and can be managed remotely. Once traffic reaches Cloudflare multiple security controls can be overlaid such as

- Cloud based network firewall ([Magic Firewall](/magic-firewall/))
- Secure web gateway ([Gateway](/cloudflare-one/policies/gateway/))

Below Is the high level traffic flow that correlates to the above diagram

> 1. Internet destined traffic will be routed to cloudflare from connected guest networks, this can be easily done with a policy based route. In most guest wifi setups devices will only be expected to generate internet bound traffic so in most cases a Policy based route referencing ANY as the destination will be sufficient. Ex Source 192.168.53.0/24 to Destination ANY next hop Cloudflare IPsec tunnel.

Reviewer: ```suggestion
1. Internet destined traffic will be routed to cloudflare from connected guest networks, this can be easily done with a policy based route. In most guest Wi-Fi setups devices will only be expected to generate internet bound traffic so in most cases a Policy based route referencing ANY as the destination will be sufficient. Ex Source 192.168.53.0/24 to Destination ANY next hop Cloudflare IPsec tunnel.
```

> 2. Once traffic reaches the cloudflare edge it will first be inspected by magic firewall. Magic firewall can be used to create network+transport layer blocks which would allow admins to restrict access to certain destination ip's or ports, a common policy could be blocking all DNS traffic not directed towards cloudflare DNS resolvers. Custom lists can be used to import existing lists customers may already have. [IDS](/magic-firewall/about/ids/) can be enabled to monitor if any guest users are attempting to launch known exploits from your Guest network. Managed threat [lists](/waf/tools/lists/managed-lists/#managed-ip-lists) allow you to use cloudflare's auto updated threat intel to block known threats like known malware repositories or botnets.

Reviewer: ```suggestion
2. Once traffic reaches the cloudflare edge it will first be inspected by magic firewall. Magic Firewall can be used to create network and transport layer blocks which would allow admins to restrict access to certain destination IP's or ports, a common policy could be blocking all DNS traffic not directed towards cloudflare DNS resolvers. Custom lists can be used to import existing lists customers may already have. [IDS](/magic-firewall/about/ids/) can be enabled to monitor if any guest users are attempting to launch known exploits from your Guest network. Managed threat [lists](/waf/tools/lists/managed-lists/#managed-ip-lists) allow you to use cloudflare's auto updated threat intel to block known threats like known malware repositories or botnets.
```

4. Traffic is now routed to the final internet destination, return traffic will be routed back through cloudflare edge and returned to the corresponding IPsec tunnel.

## Summary

By following these strategies and leveraging Cloudflare Zero Trust, organizations can offer a secure, reliable, and policy-compliant wireless experience for their guests. These measures not only safeguard networks but also enhance visibility and enable proactive threat mitigation.

> If you're interested in learning more about Gateway, or other aspects of the Cloudflare SASE platform, please visit our [reference architecture library](/reference-architecture/) or our [developer docs](/) to get started.

Reviewer: ```suggestion
If you are interested in learning more about Gateway, or other aspects of the Cloudflare SASE platform, refer to our [Reference Architecture library](/reference-architecture/) or our [Developer docs](/) to get started.
```
